packages/lab/stories/stepped-tracker/stepped-tracker.stories.tsx

```diff
@@ -198,8 +227,8 @@ export const AutoProgress: StoryFn<typeof SteppedTracker> = () => {
       style={{ width: "100%", minWidth: 600, maxWidth: 800, margin: "auto" }}
     >
       <SteppedTracker activeStep={activeStep}>
-        {steps.map(({ label, state }) => (
-          <TrackerStep state={state} key={label}>
+        {steps.map(({ label, state }, key) => (
+          <TrackerStep stage={state} key={key}>
```

Comment:
Is label not unique? and I would rather we called it index since it's the index

Reply:
Label is unique for these stories 🤷  That was just muscle-memory in action because there's no guarantee it's unique. Putting it back to `label` for now (although good point re: key/index 👍 )